Makefile

```diff
@@ -121,3 +121,8 @@ watch_css:
 sync:
 	rsync -azru --delete --stats public/ ../prism-break-static/public/
 	(cd ../prism-break-static; git add -A; git commit -m 'regenerate'; git push)
+
+# Localizations
+
+localize_%:
+	lsc ./source/functions/find-missing-localizations.ls $*
```

Comment:
Alright in that case maybe we call it from Travis based on what languages are touched in a PR, but that's a job for another issue. For this one it looks like all you need to do is add this target to the `.PHONY:` list (since isn't something that the build process is expected to output and hence there is no possibility of dependency checking or knowing whether it's 'needed' at make time).